Doc/c-api/long.rst

```diff
@@ -653,3 +653,156 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
 
    .. versionadded:: 3.12
 
+
+Export API
+^^^^^^^^^^
+
+.. versionadded:: 3.14
+
+.. c:struct:: PyLongLayout
+
+   Layout of an array of "digits" ("limbs" in the GMP terminology), used to
+   represent absolute value for arbitrary precision integers.
+
+   Use :c:func:`PyLong_GetNativeLayout` to get the native layout of Python
+   :class:`int` objects, used internally for integers with "big enough"
+   absolute value.
+
+   See also :data:`sys.int_info` which exposes similar information to Python.
```

Comment:
```suggestion
   See also :data:`sys.int_info` which exposes similar information in Python.
```

Technically "to" is correct, but it's ambiguous (is the information _exposed_ to, or _similar_ to?). "In" Python is clearer, even though it's not quite right.